### DISCUSSION TITLE
Uniquely identify a tenant and a site

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#979] [Ideas]
The idea is to extent the two tables mentioned with a GUID field to uniquely identify between installations.

Moving data between sites allows me to identify where the data has originated from.

**Comment by sbwalker** [comment]
Oqtane supports shared or isolated tenancy. Tenants are managed in the “master” database and Sites are managed in each individual tenant database. The default installation uses the shared approach where the master and tenant tables are in the same physical database, but this is just for simplicity to get the Initial system up and running. Can you explain what you are referring to by “moving data between sites”?
